MobileWallet/Screens/Settings/BackUpSettings/BackupWalletSettingsViewController.swift

```diff
@@ -146,12 +220,53 @@ extension BackupWalletSettingsViewController: UITableViewDelegate, UITableViewDa
         65
     }
 
+    func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        if tableView.numberOfSections - 1 == section && iCloudBackup.backupExists() {
+            return 50.0
+        } else {
+            return 0
+        }
+    }
+
+    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        if tableView.numberOfSections - 1 != section || !iCloudBackupsSwitcherIsOn {
+            return nil
+        }
+
+        if let lastBackupString = ICloudBackup.shared.lastBackupString {
+            let footer = UIView()
+            footer.backgroundColor = .clear
+
+            let lastBackupLabel =  UILabel()
+            lastBackupLabel.font = Theme.shared.fonts.settingsTableViewLastBackupDate
+            lastBackupLabel.textColor =  Theme.shared.colors.settingsTableViewLastBackupDate
+
+            lastBackupLabel.text = NSLocalizedString("Last successful backup: \(lastBackupString)", comment: "Settings view")
```

Comment:
Needs to be a proper key and also you can't insert a var this way with `NSLocalizedString `

Reply:
yeah, i missed that string. Thx